### DISCUSSION TITLE
[Fixed] DO NOT pacman -Syu (as of 20221118)

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#1096] [Tips]
It'll break rAudio GUI.

**Comment by rern** [comment]
Fixed.

**Comment by R101G** [comment]
Not sure about this @rern 
I updated before seeing the warning, and now the update button does not work, the USB stick shows as unmounted on the System page but works in the library, and the Android app cannot connect.

**Comment by R101G** [comment]
@rern 
What has happened is that my fixed IP has been sidelined. An en.network file has been added to /etc/systemd/network and my net.ifnames=0 entry in the /boot/cmdline.txt has gone. I restored the entry and deleted the en.network file, but I am not sure how future-proof. this is. It might be better to modify the en.network file for a fixed IP, rather than have an eth.network file.

**Reply by rern** [reply]
The latest kernel has changed ethernet device enumeration from `eth0` to `end0` and the file is `/etc/systemd/network/en.network`.
An update for the Networks page will be ready soon.

By the way, since you've already had to exclude the IP from DHCP range,  assigning a reserved IP by the router would be easier to manage.

**Comment by R101G** [comment]
@rern I prefer to use fixed IP's for my devices. Most routers keep the DHCP pool away from the lower numbers, and if I swap the router I only have to change its SSID and password, rather than reassign the all reserved addresses as well. 

I have set up the en.network file for fixed IP, and I think this should be an option on the menu.
